Dockerfile.processing_image

```diff
@@ -19,7 +19,7 @@ ADD requirements.txt requirements-base.txt
 ADD backend/corpora/dataset_processing/requirements.txt requirements-processing.txt
 RUN pip3 install -r requirements-base.txt -r requirements-processing.txt
 
-RUN wget "https://drive.google.com/u/0/uc?id=1RxbKLbVdmD40dk6DAz4CkpL0MgI4RSms&export=download" -O cellxgene-schema.tar.gz
+RUN wget "https://drive.google.com/u/0/uc?id=10JtP6-aYnU1bRJQv5dGEpfYDWndtO7H_&export=download" -O cellxgene-schema.tar.gz
```

Comment:
workaround to use cellxgene-schema 3.0.0rc10, which includes dependency updates (will be replaced with actual 3.0.0 release in requirements.txt before merging into main)